Resources/Prototypes/_LostParadise/Entities/Clothing/misc.yml

```
@@ -0,0 +1,224 @@
+  - type: Sprite
+    sprite: _LostParadise/Clothing/HalloweenDecor/Clothing/Jumpsuit/werewolf.rsi
+  - type: Clothing
+    sprite: _LostParadise/Clothing/HalloweenDecor/Clothing/Jumpsuit/werewolf.rsi
+
+- type: entity
+  parent: ClothingNeckBase
+  id: LPPClothingNeckCloakDracula
+  name: Dracula's cloak
+  description: "This cape is straight out of Transylvania of the most legendary bloodsucker in the history of mankind. It smells like garlic and aspen."
+  components:
+  - type: Sprite
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)